src/samples/java/com/box/sdk/samples/appusers/DashboardServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.box.sdk.samples.appusers;
+
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.BoxFile;
+import com.box.sdk.BoxFolder;
+import com.box.sdk.BoxItem;
+import com.box.sdk.samples.appusers.helpers.BoxHelper;
+
+
+public class DashboardServlet extends HttpServlet {
+    private String dashBoardMessage;
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException,
+            IOException {
+        if (BoxHelper.getBoxAppUserName(request) == null) { // session has timed out
```

Comment:
Not sure how to avoid this.  Each endpoint needs to verify it has a user logged in via a proper session data.  We could wrap it in a function, but since it redirects on failure I think the logic is easier to understand if it is in each servlet.